akka/src/main/scala/zio/actors/akka/AkkaTypedActor.scala

```diff
@@ -0,0 +1,41 @@
+package zio.actors.akka
+
+import akka.actor.typed
+import akka.actor.typed.ActorRef
+import zio._
+
+object AkkaTypedActor {
+
+  /**
+   *
+   * @tparam T behavior type
+   * @return reference to the typed actor in effect that can't fail and requires an akka ActorRef
+   */
+  def makeAkkaProvided[T](): URIO[ActorRef[T], AkkaTypedActorRefLocal[T]] =
```

Comment:
Why we're providing akka actor via Environment here? As it's just a proxy ActorRef, I think we can provide it via argument, like in [deprecated actorFactory in akka](https://doc.akka.io/api/akka/current/akka/actor/TypedActorFactory.html#typedActorOf[R%3C:AnyRef,T%3C:R](props:akka.actor.TypedProps[T],actorRef:akka.actor.ActorRef):R) and return it wrapped in an effect. WDYT?

Reply:
I used `zio-akka-clusters` as a reference, that's why is needed `ZIO.environment`, but you are right, is better via argument 

for example:
` def makeAkkaProvided[F[+_]](actorRef: typed.ActorRef[F[_]]): UIO[ActorRef[F]] =
    for {
      akkaActorRef <- UIO(actorRef)
    } yield new AkkaTypedActorRefLocal[F](akkaActorRef.path.toString,akkaActorRef)`